test/deploy-pipeline-lite.sh

```diff
@@ -49,8 +51,28 @@ kustomize edit set image gcr.io/ml-pipeline/inverse-proxy-agent=${GCR_IMAGE_BASE
 cat kustomization.yaml
 
 kustomize build . | kubectl apply -f -
+
 # show current info
 echo "Status of pods after kubectl apply"
 kubectl get pods -n ${NAMESPACE}
 
+if [ "$ENABLE_WORKLOAD_IDENTITY" = true ]; then
+  # Use static GSAs for testing, so we don't need to GC them.
+  export SYSTEM_GSA="test-kfp-system"
+  export USER_GSA="test-kfp-user"
+
+  yes | PROJECT_ID=$PROJECT CLUSTER_NAME=$TEST_CLUSTER NAMESPACE=$NAMESPACE \
+    ${DIR}/../manifests/kustomize/gcp-workload-identity-setup.sh
+
+  gcloud projects add-iam-policy-binding $PROJECT \
+    --member="serviceAccount:$SYSTEM_GSA@$PROJECT.iam.gserviceaccount.com" \
+    --role="roles/editor"
+  gcloud projects add-iam-policy-binding $PROJECT \
+    --member="serviceAccount:$USER_GSA@$PROJECT.iam.gserviceaccount.com" \
+    --role="roles/editor"
+
+  source "$DIR/../manifests/kustomize/wi-utils.sh"
+  verify_workload_identity_binding "pipeline-runner" $NAMESPACE
```

Comment:
Note, after a lot of rounds of trial and error, I figured out the following:
1. workload identity binding doesn't work for the first few requests no matter how long you wait for it
2. after you try to use the binding a few times, it starts to work stably

That's why I call these verification functions for each service account that's important in the test suite.

Reply:
That's interesting. Should we surface this to GKE team?

Reply:
Yes, I think so. I will send an email next week